drivers/python/rethinkdb/_import.py

```diff
@@ -793,7 +803,9 @@ def import_directory(options):
 
     spawn_import_clients(options, files_info)
 
-def table_check(progress, conn, db, table, pkey, force):
+def table_check(progress, conn, db, table, create_args, force):
+    pkey = create_args["primary_key"]
```

Comment:
Not sure I understood exactly what you meant concerning L820, but I just pushed another commit to address this.